docs/DeveloperGuide.md

```diff
@@ -50,6 +57,9 @@ This project follows [oss-generic coding standards](https://oss-generic.github.i
 
 Optionally, you can follow the [Using Checkstyle](UsingCheckstyle.md) document to configure *Intellij* to check style-compliance as you write code.
 
+### Configuring the JavaScript coding style
+Our project follows the [Airbnb Javascript Style Guide](https://github.com/airbnb/javascript), the eslint configuration file is available at the root of the project. Please run a `npm run lint -- --fix frontend/src/**/*js` from the project root directory and fix all of the eslint errors before committing your code for final review.
```

Comment:
Just remembered, users may not have installed `npm`, so might want to at least add a note and link to that installation for this.